docs/api/api.md

```diff
@@ -200,6 +194,21 @@ None.
 }
 ```
 
+### `zks_getTestnetPaymaster`
+
+Returns the address of the [testnet paymaster](../dev/zksync-v2/aa.md#testnet-paymaster): the paymaster that is available on testnets and allows to pay fees in any ERC-20 compatible tokens. 
```

Comment:
```suggestion
Returns the address of the [testnet paymaster](../dev/zksync-v2/aa.md#testnet-paymaster): the paymaster that is available on testnets and enables paying fees in ERC-20 compatible tokens. 
```

we had the allow vs. enable discussion in the past : )
feel free to ignore though